src/public_parameters/registry.rs

```diff
@@ -56,7 +56,7 @@ impl Registry {
             let pp = default(lang);
             disk_cache
                 .set(key, &*pp)
-                .tap_ok(|_| eprintln!("Writing public params to disk-cache: {}", lang_key))
```

Comment:
This println should stay until we have a better plan, such as a `dbg!()` output

Reply:
Added it back in.